text/0107-construct-library-module-lifecycle.md

```diff
@@ -0,0 +1,144 @@
+    * **Documentation** - update module’s README with explanations and code snippets that illustrate how to use new constructs as they are developed
+    * **Publish aws-xyz-experimental package** - deliver experimental constructs in their own package to clearly communicate module maturity. This package is tagged with Stability:Experimental
+    * **Observe** - monitor module utilization and GitHub issues filed against the experimental module
+        * feature requests and bugs filed against the experimental module are triaged, those that we decide to include in the module are linked in the tracking issue to indicate they are in scope
+    * **Repeat** - These activities are repeated until the module owner decides the module is feature complete and the API is ergonomic. At the end of the experimental stage, the construct library module’s GitHub tracking issue will have a description of the service, a list of targeted use cases, and a list of L2 constructs required to satisfy the use cases.
+* **Exit Criteria**:
+    * **Feature complete** - The module owner declares the module feature complete for targeted use cases, and the API is ergonomic. 
+        >**Note**: Feature complete does not mean 100% AWS service feature coverage, instead, it means the module owner thinks we’ve implemented the minimum features required for a delightful developer experience for modeling the service infrastructure as code.
+     * **Successful API review** - conduct an API review/developer experience demo and invite relevant stakeholders
+        * use module's README as the API review guide
+        * address any concerns about API ergonomics
+        * review all lint exceptions to ensure they are acceptable for dev preview
+        * ensure at least 90% unit test coverage for all implemented L2 constructs
+        * if the API/DevEx is rejected, make a list of necessary remediation tasks and update the module's tracking issue 
+
+## Stage 2 - Developer Preview 
+
+The goal of developer preview is to deliver a release candidate with a stable API to conduct user acceptance testing and accumulate sufficient usage to declare a module is ready for general availability. We will only make breaking changes to developer preview modules when required to address unforeseen customer use cases or issues. Since the potential for breaking changes still remains while in Developer Preview, the package name will keep the “-experimental” suffix during this stage.
+
+* **Stage 2 activities**:
+    * **Update Roadmap** - move to the CDK public roadmap “Developer Preview” column to indicate we need customer feedback on the module’s implementation
+    * **Observe** - monitor module utilization and GitHub issues filed against the experimental module
+    * **Quality Assurance** - fix all P0 bugs as they are  discovered and review all P1 bugs, making a clearly communicated fix/defer decision for each one 
+* **Exit Criteria**:
```

Comment:
How about something around the AWS Lint checks?

Reply:
whoops... I see AWS Lint called out as part of the bar to get to Dev Preview.  It may make sense to explicitly still include here as something we double check to make sure nothing has snuck in, and to catch anything already in DevPreview before this process started.

Reply:
OK, I'll explicitly callout AWS Lint checks. Is it safe to assume that lint checks happen in every PR review?  i.e. if a new PR is cut during developer preview to address an issue, any lint exceptions are also reviewed at that time?